### PR TITLE
[stable/redis] Add myself as maintainer

### PR DESCRIPTION
**Pull request by desaintmartin**
As an owner of several k8s production clusters heavily using redis chart, I'd like to help in maintaining / reviewing this chart.

A few PRs I've done: https://github.com/helm/charts/pulls?utf8=%E2%9C%93&q=redis+is%3Apr+author%3Adesaintmartin+

cc @carrodher  @juan131 @tompizmor @prydonius @sameersbn

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped

### CODE REVIEW

**Comment by prydonius** [comment]
thanks @desaintmartin, would be awesome to have you help!

**Comment by carrodher** [comment]
Welcome @desaintmartin 😉 
For me it is ok, but it seems that circleci linter doesn't think the same, can you take a look at the issue? It seems you need to set your GH username instead of your real life name

**Comment by desaintmartin** [comment]
Of course! Fixed.

File: stable/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.11.1
+version: 1.11.2
```
Comment (carrodher) [inline stable/elasticsearch/Chart.yaml]: Unwanted change
```suggestion
version: 1.11.1
```

**Comment by desaintmartin** [comment]
Requesting to be a maintainer, doing two errors, nice.
Fixed.

**Pull request merged**